google-authenticator.php

```diff
@@ -312,15 +312,15 @@ function user_setup_page() {
 		),
 		array(
 			'text' => __( 'iOS: Google Authenticator', 'google-authenticator' ),
-			'link' => 'https://itunes.apple.com/app/authy/id494168017',
+			'link' => 'https://itunes.apple.com/us/app/google-authenticator/id388497605',
```

Comment:
I think it's better to leave the original link, as not everyone is using the US store.

Reply:
sorry, I was wrong. original link is better.

Reply:
I think based on the reviews, the authy one is better, or we could do the same as for Android and link to both google authenticator and authy and let the end user decide

Reply:
at source code, iOS authy link is exits.

`array(
			'text' => __( 'iOS: Authy', 'google-authenticator' ),
			'link' => 'https://itunes.apple.com/app/authy/id494168017',
		),`

and, link text is "iOS: Google Authenticator". so i think 'iOS: "Google" Authenticator" link is shoud not link to 
the "authy"...

if provide 2 links, i think end user can select which.

Reply:
you are correct @kassyi 

'link' => 'https://itunes.apple.com/app/google-authenticator/id388497605',

Will suffice